pkg/proxy/winkernel/proxier.go

```diff
@@ -128,12 +128,21 @@ type endpointsInfo struct {
 	refCount   uint16
 }
 
+//Uses mac prefix and IP address to return a mac address
+//This ensures mac addresses are unique for proper load balancing
+func conjureMac(macPrefix string, ip net.IP) string { 
+	p := ip
+	p4 := p.To4();
```

Comment:
Please add a check of nil in case panic in the following line.

nit: `p:=ip` is not required here, just use `ip.To4()`